### PR TITLE
Update mooRainbow to version 1.4.

### PR DESCRIPTION
**Pull request by realityking**

### CODE REVIEW

File: media/system/css/mooRainbow.css [hunk 1]
```diff
@@ -83,7 +83,7 @@ label {
 	margin-top: 190px;
 	margin-left: 315px;
 }
-span.moor-ballino { /* Style hue ° (degree) !! */
+span.moor-ballino { /* Style hue Âº (degree) !! */
```
Comment (LouisLandry) [inline media/system/css/mooRainbow.css]: Looks like you may have gotten a junk character in here.

**Comment by infograf768** [comment]
file has to be encoded as UTF8

**Comment by infograf768** [comment]
and
-label {
-   font-family: mono;
  -}

has to be taken off as it has unwanted display results in CMS back-end.

**Comment by realityking** [comment]
I took of the first commit so we can just deal with the update.

**Pull request merged**